pkg/api/controller.go

```diff
@@ -3006,6 +3011,66 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email []string, token string) error {
+	err := c.Emailer.SendEmail(email, token, token, []string{})
+	if err != nil {
+		c.Logger.
+			WithError(err).
+			WithFields(logging.Fields{
+				"email": email,
+			}).Debug("Error sending email")
+		return err
+	}
+	c.Logger.WithFields(logging.Fields{
+		"email": email,
+	}).Info("Succefully sent email")
+	return err
+}
+
+func (c *Controller) PasswordForgot(w http.ResponseWriter, r *http.Request, body PasswordForgotJSONRequestBody) {
+	user, err := c.Auth.GetUserByEmail(r.Context(), body.Email)
+	if err != nil {
+		writeError(w, http.StatusForbidden, http.StatusText(http.StatusForbidden))
```

Comment:
I think this is wrong, here and on l. 3037 you need to send 401 (Unauthorized).  See [403 Forbidden](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403), [401 Unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401).

Confusingly, "forbidden" in HTTP generally corresponds to authorization failure ("I know who you are and I'm not going to let you do it"), while "unauthorized" generally corresponds to authentication failure ("you didn't prove to me who you are").

Reply:
Your right. Fixed